File: bin/ans/achen_easyprob.cpp
/*
PROG: easyprob
LANG: C++
*/

#include <iostream>
#include <cstdio>
#include <cstdlib>

using namespace std;

int main()
{
    int N;
    scanf( "%d", &N);
    int total = 0, x;
    for(int i = 0; i < N; i++)
    {
        scanf("%d", &x);
        total += x;
    }
    printf("%d\n", total);

    return 0;
}

File: bin/ans/achen_easyprob.java
/*
PROG: easyprob
LANG: JAVA
*/

import java.util.*;
import java.io.*;

public class easyprob
{
    public static void main(String[] alex) throws IOException
    {
        //BufferedReader w = new BufferedReader(new FileReader("easyprob.in"));
        //PrintWriter out = new PrintWriter(new BufferedWriter(new FileWriter("easyprob.out")));

	Scanner sc=new Scanner(System.in);
        int sum = 0;
        int N = Integer.parseInt(sc.nextLine());
        for(int i = 0; i < N; i++)
            sum += Integer.parseInt(sc.nextLine());
        System.out.println(sum);
        //out.close();

        System.exit(0);
    }
}
